### DISCUSSION TITLE
How do you create a new color variable?

### DISCUSSION THREAD

**Discussion by BenJackGill** [quasarframework/quasar#12286] [General - Components / Directives / etc] [answered]
There are various colors we can set inside the `quasar.variables.scss` file (`$primary`, `$secondary`, etc) but how do we create a brand new color variable (`$my-new-color`) and have it work inside props?

For example: `<q-btn color="my-new-color" text-color="my-new-color" />`.

**Comment by pikil** [comment, marked as answer]
This one should help:
https://quasar.dev/style/color-palette#adding-your-own-colors

**Reply by BenJackGill** [reply]
Simple! Thanks :)

For anyone else, to get a global color you need to use something like this in your `quasar.variables.scss` file.

```
$my-new-color: #333333;

.text-my-new-color {
  color: $my-new-color !important;
}

.bg-my-new-color {
  background: $my-new-color !important;
}
```
